ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test/unixext_test.ml

```diff
@@ -281,8 +281,26 @@ let test_select =
   in
   true
 
+let test_direct =
+  let gen = Gen.unit in
+  Test.make ~count:1 ~name:__FUNCTION__ gen @@ fun _ ->
+  let fn = "test_file_direct" in
+  let buf = Bytes.make 10000 'x' in
+  Unixext.Direct.with_openfile fn [Unix.O_RDWR; Unix.O_CREAT] 0o600 (fun f ->
```

Comment:
Do you plan on using this module in a future PR?

There is a lot of dead code in XAPI unfortunately, and in this case the entire Unixext.Direct module appears to be dead code. If I drop the entire module from the interface and impl, then XAPI still builds.